src/index.js

```diff
@@ -51,6 +52,7 @@ module.exports = {
       },
       rules: {
         'jsx-a11y/alt-text': 'error',
+        'jsx-a11y/anchor-ambiguous-text': 'error',
```

Comment:
adding a new rule to a config is a breaking change, so we can't do that here.
```suggestion
        'jsx-a11y/anchor-ambiguous-text': 'off', // TODO: error
```

Reply:
Ah, I see. This portion was auto-generated with the new rule script (i.e. `./scripts/create-rule.js`), so maybe that needs to be updated too?

Reply:
agreed